python_modules/dagster/dagster/_core/storage/dagster_run.py

```diff
@@ -428,6 +430,56 @@ def get_root_run_id(self) -> Optional[str]:
     def get_parent_run_id(self) -> Optional[str]:
         return self.tags.get(PARENT_RUN_ID_TAG)
 
+    @cached_property
+    def well_known_tags(self) -> Dict[str, str]:
```

Comment:
thinking through naming:
* i think the fact that `DagsterRun` has `tags` already means we should include something in the name that tries to clarify that these are not those / do not come from those 
* "well known" is a bit vague to me. From your PR summary "execution" / "orchestration" stand out. "system" and "context" come to mind for me too.

so like something in the direction of 
`derived_system_tags` 
`derived_execution_tags` 
`computed_origin_tags` 
`create_execution_tags` 
or avoid the word tags 
`system_info`
`origin_context`
`execution_labels`